### PR TITLE
expose timezone identifier using getTimezone() function, same changes…

### PR DESCRIPTION
**Pull request by mdulat**
… as applied to hybrid

### CODE REVIEW

File: d2l-localize-behavior.js [hunk 1]
```diff
@@ -88,7 +92,7 @@ D2L.PolymerBehaviors.LocalizeBehaviorImpl = {
 				} else if (mutation.attributeName === 'data-intl-overrides') {
 					this.__overrides = this._tryParseHtmlElemAttr('data-intl-overrides', {});
 				} else if (mutation.attributeName === 'data-timezone') {
-					this.__timezone = this._tryParseHtmlElemAttr('data-timezone', {name: ''}).name;
+					this.__timezone = this._tryParseHtmlElemAttr('data-timezone', {name: '', identifier: ''});
```
Comment (mdgbayly) [inline d2l-localize-behavior.js]: should be assigning to __timezoneObject?

Comment (mdulat) [reply d2l-localize-behavior.js]: oops

**Review by dlockhart — APPROVED** [review]
Looks good. Next time, in theory you should be able to just re-run the Polymer 3 migration on the hybrid branch, apply the patch and end up in the same place. In theory.

**Comment by mdulat** [comment]
@dlockhart in theory :) I tried that first, but it did not work, ended up being easier to re-apply the changes manually, as they were quite small.

**Pull request merged**